applications/nrf_desktop/configuration/nrf54lm20dk_nrf54lm20a_cpuapp/sysbuild.conf

```diff
@@ -6,9 +6,12 @@
 ################################################################################
 
 SB_CONFIG_BOOTLOADER_MCUBOOT=y
-SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP=y
+SB_CONFIG_MCUBOOT_MODE_RAM_LOAD=y
```

Comment:
I wonder if we shouldn't leave one configuration that uses `SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP` to have both DFU methods support for nRF54LM20. Thanks to that we could have configuration that is not "experimental" (but has report rate issues in USB HS mode).

Reply:
- `ram_load`
- `release_ram_load`